app/src/main/java/tech/relaycorp/letro/utils/i18n/Locale.kt

```diff
@@ -0,0 +1,23 @@
+package tech.relaycorp.letro.utils.i18n
+
+import java.util.Locale
+
+fun Locale.normaliseString(): String {
```

Comment:
Just in case, we have tech.relaycorp.letro.utils.ext for extension functions. You can do it now or I was going to sort classes to the packages, because now there is a mess, so it's not critical, I can do it later

Reply:
I missed that package, but to be honest, I'd rather group things by semantics, not by what type of Kotlin _thing_ it is. If we create a package for extension functions, would we also create a package for classes and another for standalone functions? I don't think we'd do that.

Reply:
I second that